doc/howto/import_machines_to_instances.md

```diff
@@ -34,6 +34,30 @@ The tool can create both containers and virtual machines:
   It is also not possible to create a virtual machine from the physical machine that you are using to do the migration, because the migration tool would be using the disk that it is copying.
   Instead, you could provide a bootable image, or a bootable partition or disk that is currently not in use.
 
+   ```{tip}
+   If you want to convert Windows VM from a foreign hypervisor (not from QEMU/KVM with Q35/virtio-scsi),
+   then you also need to install virtio-win drivers to your Windows. Otherwise, your VM won't boot.
+   ```
+
+   <details>
+   <summary>Expand to see how to integrate virtio-win drivers to your Windows VM</summary>
+
+   1. Install virt-v2v version >= 2.3.4 (this is the minimal version that supports `--block-driver` option)
+   1. Install virtio-win package or download iso directly from [virtio-win.iso](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso) and put to `/usr/share/virtio-win`
+   1. You may also need to install [rhsrvany](https://github.com/rwmjones/rhsrvany#full-example)
+   1. Now we are ready to use `virt-v2v` to convert images from a foreign hypervisor to `raw` images for LXD
+
+   ```
+   # Example 1. Convert a vmdk disk image to a raw image suitable for lxd-migrate
+   sudo virt-v2v --block-driver virtio-scsi -o local -of raw -os ./os -i vmx ./test-vm.vmx
+
+   # Example 2. Convert a QEMU/KVM qcow2 image and integrate virtio-scsi driver
+   sudo virt-v2v --block-driver virtio-scsi -o local -of raw -os ./os -if qcow2 -i disk test-vm-disk.qcow2
+   ```
+
+   Now you can find a resulting image in the `os` directory and use it with `lxd-migrate` on the next steps.
+   </details>
+
```

Comment:
`````suggestion
   ````{tip}
   If you want to convert a Windows VM from a foreign hypervisor (not from QEMU/KVM with Q35/`virtio-scsi`),
   you must install the `virtio-win` drivers to your Windows. Otherwise, your VM won't boot.

   <details>
   <summary>Expand to see how to integrate the `virtio-win` drivers to your Windows VM</summary>

   1. Install `virt-v2v` version >= 2.3.4 (this is the minimal version that supports the `--block-driver` option).
   1. Install the `virtio-win` package, or download the [`virtio-win.iso`](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso) image and put it into the `/usr/share/virtio-win` folder.
   1. You might also need to install [`rhsrvany`](https://github.com/rwmjones/rhsrvany#full-example).
   
   Now you can use `virt-v2v` to convert images from a foreign hypervisor to `raw` images for LXD:

   ```
   # Example 1. Convert a vmdk disk image to a raw image suitable for lxd-migrate
   sudo virt-v2v --block-driver virtio-scsi -o local -of raw -os ./os -i vmx ./test-vm.vmx

   # Example 2. Convert a QEMU/KVM qcow2 image and integrate virtio-scsi driver
   sudo virt-v2v --block-driver virtio-scsi -o local -of raw -os ./os -if qcow2 -i disk test-vm-disk.qcow2
   ```

   You can find the resulting image in the `os` directory and use it with `lxd-migrate` on the next steps.
   </details>

````
`````

Reply:
Let's put the instructions inside the tip - tbh I'm not sure how it'll look in Sphinx, but it's worth a try. :)

Reply:
I think it might also help to be explicit about where to install the tools. They all need to be installed on the host, is that correct?
In the intro, it sounds like you need to install them on Windows.

Reply:
>I think it might also help to be explicit about where to install the tools. They all need to be installed on the host, is that correct?

yeah, you need to put `virtio-win.iso` and stuff on the host, but then these files will be used by `virt-v2v` to install that on Windows

Reply:
OK, thanks for clarifying - I updated the suggestion above to make this clear.

Reply:
I've pushed your version. Thanks!